drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -469,7 +462,9 @@ function DragAndDropEditBlock(runtime, element, params) {
                             if (name.length > 0 || imageURL.length > 0) {
                                 var data = {
                                     displayName: name,
-                                    zone: $el.find('.zone-select').val(),
+                                    zones: $.map($el.find('.zone-checkbox:checked'), function(c){
```

Comment:
@haikuginger Yes, that's exactly what I meant :) You won't need the `var` though; you can just add the declaration to the existing list of declarations above the `if` (keeping the order in which the fields appear):

```js
                            var $el = $(el),
                                name = $el.find('.item-text').val(),
                                $zones = $el.find('.zone-checkbox:checked'),
                                imageURL = $el.find('.item-image-url').val(),
                                imageDescription = $el.find('.item-image-description').val();

                            if (name.length > 0 || imageURL.length > 0) {
                                var data = {
                                    displayName: name,
                                    zones: $.map($zones, function(zone) {
                                        return zone.value;
                                    }),
                                    // ...
```